engine/src/main/java/io/zeebe/engine/processor/workflow/CatchEventBehavior.java

```diff
@@ -83,12 +83,13 @@ public void subscribeToEvents(
     // if all subscriptions are valid then open the subscriptions
     for (final ExecutableCatchEvent event : events) {
       if (event.isTimer()) {
+        final Timer timer = event.getTimerFactory().apply(expressionProcessor, context.getKey());
```

Comment:
What happens if we have multiple timer/message subscriptions and one of the timer expressions fails?

If an incident is created then no other action should be applied to the workflow instance (e.g. don't open other subscriptions, don't go to ELEMENT_ACTIVATED state)